cypress/e2e/Probes/probes.spec.cy.ts

```diff
@@ -9,9 +9,15 @@ import { ProbesCRUDTests } from "./tests/probesPageTests";
 describe("Probes section", () => {
   before(() => {
     setup();
+    cy.snowflakeSql("deleteProbes");
+  });
+
+  after(() => {
+    cy.snowflakeSql("deleteProbes");
   });
 
   beforeEach(() => {
+    cy.snowflakeSql("deleteProbes");
```

Comment:
@susannahmarcus , can you please remind me. Does this delete all the predefined probes as well ? (Do predefined probes _live_ in internal.probes  together with user defined probes?)

Reply:
it does, but that said, do we need the predefined probes within the testing env?

Reply:
well ... chicken and egg problem. 
(a) we need to make sure that python tests verify that pre-defined labels where successfully imported (not sure we have this test, got to verify)
(b) on every initial installation users need to see all the labels and probes in the UI 

we can deal with it later ... I think.